Client/Application/LeanplumIntegration.swift

```diff
@@ -132,23 +132,25 @@ class LeanplumIntegration {
 
         Leanplum.start(userAttributes: userAttributesDict)
 
-        track(eventName: LeanplumEventName.openedApp)
-
-        // Only drops Leanplum event when a user has installed Focus (from a fresh state or a re-install)
-        if profile?.prefs.boolForKey(PrefsKeys.HasFocusInstalled) == canInstallFocus() {
-            profile?.prefs.setBool(!canInstallFocus(), forKey: PrefsKeys.HasFocusInstalled)
-            if !canInstallFocus() {
-                track(eventName: LeanplumEventName.downloadedFocus)
+        Leanplum.onStartResponse({ _ in
```

Comment:
This `onStartResponse` should either be wrapped in an `.shouldSendToLP() {` or moved into `LeanplumIntegration` where that is done for every call to Leanplum.

Does this delay startup time?

We've had a lot of concurrency problems with the profile being opened/accessed from different threads. So before we approve this we need to understand on what thread that `onStartResponse` callback is executed.

Cc to @justindarc - I want to be sure that is is safe to do.